scripts/convert-to-geomdat/convert-to-geomdat.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+
+# This script automatically converts a restart file to a Geometry.dat with AIMS runs.
+# It assumes that you have velocities in the restart file.
```

Comment:
A script definition is usually placed at the top of the file inside """triple quotes""" just like a docstring. Let's make that change!